### DISCUSSION TITLE
undefined is not an object (evaluating '(0, _mobxStateTree.getRoot)(self).

### DISCUSSION THREAD

**Discussion by ankit5174** [mobxjs/mobx-state-tree#1720] [Q&A]
@team can anyone help me with this error  undefined is not an object (evaluating '(0, _mobxStateTree.getRoot)(self).requestStore.withRequest. I am exposing withRequest action from requestStore as an extension

**Comment by EmilTholin** [comment]
Hi @ankit5174!

Do you think you could include a minimal reproducible example with the error you are facing? It's hard to say what might be wrong without seeing some code.
